### ISSUE TITLE
Monster hit points in info screen always appear maxed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#117]
```
What steps will reproduce the problem?
1. Injure an actor
2. Push the info button below the attack button while in combat
3. Look at the hit points bar and the numeric current/maximum hit points count.
OR
2. Disengage from combat
3. Re-engage and view its info from the button in the pre-combat dialog.

One would expect the damage already inflicted to be visible in the hit points 
bar, e.g. 17/25. Instead it always shows 25/25.


```

Original issue reported on code.google.com by `Hoh...@gmail.com` on 3 Jan 2011 at 12:09

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. Yes, this is a bug.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 8:07

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved by r147. Thanks to Scott Lund for the bugfix!
```

Original comment by `oskar.wi...@gmail.com` on 30 Aug 2011 at 9:00
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
